modules/table/tech-notes/rebalance.md

```diff
@@ -96,6 +96,7 @@ metastoreInvoke: \\ atomic
         partition.assignments.pending = empty
     else:
         partition.assignments.pending = partition.assignments.planned
+        remove(partition.assignments.planned)
```

Comment:
Hmm, is that really all changes we should introduce in order to match design and current actual implementation?